api/turing/cluster/servicebuilder/fluentd.go

```diff
@@ -68,7 +70,7 @@ func (sb *clusterSvcBuilder) NewFluentdService(
 			LivenessHTTPGetPath:   fluentdHealthCheckPath,
 			ReadinessHTTPGetPath:  fluentdHealthCheckPath,
 			ProbeInitDelaySeconds: 10,
-			Labels:                buildLabels(project, envType, routerVersion.Router),
+			Labels:                buildLabels(project, routerVersion.Router),
```

Comment:
If `envType` is no longer required to be set from this place (which makes sense to me), should we also remove it from the `NewFluentdService` function signature now?